docs/reference/query-dsl/simple-query-string-query.asciidoc

```diff
@@ -31,8 +31,9 @@ The `simple_query_string` top level parameters include:
 |`fields` |The fields to perform the parsed query against. Defaults to the
 `index.query.default_field` index settings, which in turn defaults to `*`. `*`
 extracts all fields in the mapping that are eligible to term queries and filters
-the metadata fields. There is a limit of no more than 1024 fields being queried
-at once.
+the metadata fields. There is a limit on the number of fields that can be queried
+at once which is defined by the `indices.query.bool.max_clause_count` <<search-settings>>
+which defaults to 1024.
```

Comment:
Can you also add a note in the `multi_match` query ?

Reply:
np